src/linked_list.py

```diff
@@ -52,12 +63,15 @@ def remove(self, remove_node):
 
     def display(self):
         """Return the tuple of all values in linked list."""
-        new_list = [self.head_node.contents]
-        current_node = self.head_node
-        while current_node.next_node is not None:
-            current_node = current_node.next_node
-            new_list.append(current_node.contents)
-        return tuple(new_list)
+        if self.length == 0:
+            return None
+        else:
+            new_list = [self.head_node.contents]
+            current_node = self.head_node
+            while current_node.next_node is not None:
+                current_node = current_node.next_node
+                new_list.append(current_node.contents)
+            return tuple(new_list)
```

Comment:
This is really good and really close; however the assignment specs say:

display() will return a unicode string representing the list as if it were a Python tuple literal: “(12, ‘sam’, 37, ‘tango’)”